src/guides/v2.3/extension-dev-guide/plugins.md

```diff
@@ -485,6 +485,33 @@ Plugins can be disabled in a `di.xml` file. To disable a plugin, set the `disabl
 
 where `ProcessPaymentConfiguration` is the name of the plugin declared in the `vendor/magento/module-payment/etc/frontend/di.xml`.
 
+Note, if you need to disable a particular plugin in your custom module then you have to use the same class name which defined in the type `name` attribute.
+
+For example, if you want to disable a `bindAttributeIndexerMassUpdate` plugin:
+
+```xml
+<type name="\Magento\Catalog\Controller\Adminhtml\Product\Action\Attribute\Save">
+    <plugin name="bindAttributeIndexerMassUpdate"
+            type="\Magento\GoogleShoppingAds\Plugin\Catalog\Controller\Adminhtml\Product\Action\Attribute\Save\BindAttributeIndexer"/>
+</type>
+```
+
+then you should use the next approach
+
+```xml
+<type name="\Magento\Catalog\Controller\Adminhtml\Product\Action\Attribute\Save">
```

Comment:
Is there any difference between this example and the existing example above your change? You always have to use the same class name, no?

Reply:
Hi, @dobooth !

`\Magento\Catalog\Controller\Adminhtml\Product\Action\Attribute\Save` is not the same as `Magento\Catalog\Controller\Adminhtml\Product\Action\Attribute\Save`.

It is the same if you compare "Donald" and "Alex".

Reply:
The example in L480 says to add the `disabled='true'` attribute to disable the plugin. It seems that the example here on L502 is doing the same thing.  If there is no difference in what we are showing, there is no need to duplicate the example. And we can probably just make a note about the importance of the leading slash. A code example of a leading slash is likely overkill.

Reply:
@dobooth , let me please clarify.

The idea of my PR is: show developer a case when a plugin can be set on a particular class with a leading `backslash`.

In the example I've used a real code of real module.

For example we have a class `Magento\Catalog\Controller\Adminhtml\Product\Action\Attribute\Save`.

Developer can define a plugin for this class as:

```
<type name="Magento\Catalog\Controller\Adminhtml\Product\Action\Attribute\Save">
  <plugin name="plugin_a" ....>
....
```

or 

```
<type name="\Magento\Catalog\Controller\Adminhtml\Product\Action\Attribute\Save">
  <plugin name="plugin_b" ....>
....
```

So, if you want to disable `plugin_a` you have to call `Magento\Catalog\Controller\Adminhtml\Product\Action\Attribute\Save`. This is like an alias.

So, this is the important notice for developers which want to disable a plugin. They should refer to the original plugin declaration.

Thank you!

Reply:
I see. You can define a plugin two ways: with the leading slash or without. If you want to disable the plugin, you must use the same way to disable as it was defined.

Reply:
@dobooth , correct :)

Reply:
Good. Let me know if my changes are clear and correct.

Reply:
@dobooth , regarding to a PR purpose your changes are valid.

**Approved**

Thank you ;)